CruzHacks/cruzhacks-2021-website
# Prizes component.

Problem
=======
As a hacker, 
I want to see the available prices
So I can get excited about them!

[Link to Asana Ticket](https://app.asana.com/0/1197738625416321/1199333062665014)

Solution
========
The layout is similar to the Milestones component, expect this one doesn't have any images/graphics associated with it. ~~I had to copy over some of the mixins/variables from #91, but they will be removed after that PR is pulled in.~~

In the figma design, the first half of the Prizes section belongs to the Milestones section. I had to use separate backgrounds for the two sections to not deal with negative margins. The integrated design looks like this: 
<img src="https://user-images.githubusercontent.com/20177171/101318376-14d67600-3815-11eb-9840-8b3ca6f4f7da.png" width="500px" />

I don't see a clear way to share a background with another component without messing up the spacing somehow, but feedback / suggestions would be greatly appreciated. 

Change summary:
---------------
* Prizes component. https://github.com/CruzHacks/cruzhacks-2021-website/commit/24e95233070d4b2a6a37529e67ae9fe9e5472be8

Steps to Verify:
----------------
1. Checkout branch and run `yarn run start` to start a local web server.
2. Verify the Prizes section is present and responsive.
